repos/fdbt-site/src/components/ProductGroupCard.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { ReactElement } from 'react';
+import { GroupOfProducts } from '../interfaces';
+
+interface ProductGroupCardProps {
+    groupDetails: GroupOfProducts;
+    index: number;
+    deleteActionHandler?: (id: number, name: string) => void;
+    defaultChecked: boolean;
+}
+
+const ProductGroupCard = ({
+    groupDetails,
+    index,
+    deleteActionHandler,
+    defaultChecked,
+}: ProductGroupCardProps): ReactElement => {
+    const { name, id } = groupDetails;
+
+    return (
+        <div className="card" id={`product-group-${index}`}>
+            <div className="card__body">
+                {deleteActionHandler ? (
+                    <div className="card__actions">
+                        <ul className="actions__list">
+                            <li className="actions__item">
+                                <a
+                                    className="govuk-link govuk-!-font-size-16 govuk-!-font-weight-regular"
+                                    href={`/searchOperators?id=${id}`}
+                                >
+                                    Edit
+                                </a>
+                            </li>
+
+                            <li className="actions__item">
+                                <button
+                                    className="govuk-link govuk-!-font-size-16 govuk-!-font-weight-regular actions__delete"
+                                    onClick={() => deleteActionHandler(id, name)}
+                                >
+                                    Delete
+                                </button>
+                            </li>
+                        </ul>
+                    </div>
+                ) : (
+                    <div className="govuk-radios">
+                        <div className="govuk-radios__item card__selector">
+                            <input
+                                className="govuk-radios__input"
+                                id={`product-group-${index}-radio`}
+                                name="productGroupId"
+                                type="radio"
+                                value={id}
+                                aria-label={name}
+                                defaultChecked={defaultChecked}
+                            />
+                            {/* eslint-disable-next-line jsx-a11y/label-has-associated-control */}
+                            <label className="govuk-label govuk-radios__label" />
+                        </div>
+                    </div>
+                )}
+
+                <h4 className="govuk-heading-m govuk-!-padding-bottom-4">{name}</h4>
+
+                <p className="govuk-body-s govuk-!-margin-bottom-2">{groupDetails.productIds.length} products</p>
```

Comment:
If you mean can we add some logic to the 's' on the end, yeah ill add that.

Reply:
or as per the design ... we need to show them the products name ...

Reply:
Spoke to Steve - it makes no sense to start listing products names because there could be 100 products in a group, and showing the first 2-3 product names wont help.